.github/workflows/run_kms_ibm_kp_test.yml

```diff
@@ -6,9 +6,8 @@ on: [push, pull_request]
 jobs:
   run-kms-ibm-kp-test:
     env:
-      IBM_INSTANCE_ID: ${{ secrets.IBM_INSTANCE_ID }}
-      IBM_SERVICE_API_KEY: ${{ secrets.IBM_SERVICE_API_KEY }}
-      IBM_CUSTOMER_ROOT_KEY: ${{ secrets.IBM_CUSTOMER_ROOT_KEY }}
+      IBM_KP_INSTANCE_ID: ${{ secrets.IBM_INSTANCE_ID }}
```

Comment:
@baum can we make this a manual workflow that is triggered from a PR workflow? do you think we can solve the secret accessibility issue like that?

Reply:
According to @leseb there are two options:

1. Create a PR from the branch of  `noobaa/noobaa-operator` repository, instead of a fork
2.  Run this PR periodically out of daily or something

The integration test runs OK in the fork, here is [the link to this change run](https://github.com/baum/noobaa-operator/runs/4810598584?check_suite_focus=true).